lib/galaxy/auth/__init__.py

```diff
@@ -66,7 +66,8 @@ def check_auto_registration(self, trans, login, password, no_password_check=Fals
                     try:
                         auth_return = parse_auth_results(trans, auth_results, options)
                     except Conflict:
-                        break
+                        log.exception(Conflict)
```

Comment:
This is in a for loop, I'd rework this. Also the exception should have a message not just log a class name. I'm going to mark this as "work in progress" waiting on those changes. We're (the committers) are unsure about whether this should be logged at all though.

Reply:
Maybe I should mention that the 'to string'  ("__str__") operator is overridden in the base 'MessageException' class which 'Conflict' derive from.  Thus the error message is log, not the class name as it might seems at first.

Reply:
`Conflict` is still only the class, not the actual exception object, which you can get with e.g. `except Conflict as e:`

Reply:
Good catch. I have updated the code.